wormhole/module.py

```diff
@@ -180,17 +180,21 @@ async def on_message(self, message: discord.Message):
                 "Missing permissions to delete message.",
             )
 
-        formatted_message = await self._message_formatter(message)  # Format message
+        formatted_message_parts = utils.text.smart_split(
+            await self._message_formatter(message),
+            mark_continuation="***Pokračování***\n",
```

Comment:
The continuation has to be translated through popie (_utx/itx, "**Continuation**")